doc/source/configuration/vault.rst

```diff
@@ -210,6 +210,20 @@ Enable the required TLS variables in kayobe and kolla
 
 3. Deploy backend and internal TLS
 
+   .. warning::
+
+      It is important that you are only using admin endpoints for keystone. If
+      any admin endpoints exist for other services, they must be deleted e.g. 
+
+      .. code-block::
+
+         openstack endpoint list --interface admin -f value | \
+         awk '!/keystone/ {print $1}' | xargs openstack endpoint delete
+      
+      The services using these endpoints might also need restarting in order to
+      apply the changes. In particular, the ``nova_compute`` container has had
+      problems with this in the past. 
```

Comment:
This shouldn't be necessary if the endpoints are removed before the change is rolled out, because the services get restarted during the rollout.

Reply:
removed